config/locales/provider_interface/candidate_pool_invites.yml

```diff
@@ -17,6 +17,25 @@ en:
           no_candidates: There are no invited candidates that match your filters
           application_received: Application received
           invited: Invited
+      not_seen:
+        index:
+          title: New candidates
```

Comment:
To be consistent with the other titles, i think this should be Find candidates - New (the other ones are Find candidates - All, and Find candidates - Invited)